SConstruct

```diff
@@ -1896,12 +1911,10 @@ def postBuildMessage(target, source, env):
     print("- To run the test suite, type 'scons test'.")
     print("- To list available tests, type 'scons test-help'.")
     if env['googletest'] == 'none':
-        print("  WARNING: You set the 'googletest' to 'none' and all it's tests will be skipped.")
+        print("  WARNING: You set the 'googletest' to 'none' and all its tests will be skipped.")
```

Comment:
Should this be issued by `logger`? (Although the same applies to the surrounding lines, so replacing things just here would be odd.)

Reply:
Yes, it should. I just couldn't help fixing this typo 🤪 I'm still hoping to eventually be able to refactor this file to use those features I added in `buildutils.py`. I think the changes from #1137 are a great step in the right direction too!